services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -2465,6 +2467,19 @@ public void updateStore(String clusterName, String storeName, UpdateStoreQueryPa
 
       setStore.storagePersona = personaName.map(addToUpdatedConfigList(updatedConfigsList, PERSONA_NAME)).orElse(null);
 
+      // Check whether the passed param is valid or not
+      if (latestSupersetSchemaId.isPresent()) {
+        if (latestSupersetSchemaId.get() != SchemaData.INVALID_VALUE_SCHEMA_ID) {
```

Comment:
is the goal here to reset super schema id?  do you think there will be a case where user want to set superset schema id to a previous valid schema id?

Reply:
The goal of this PR is to offer a way to update superset schema id, and regarding which schema id to set depends on the use case.
For the rollback of partial update feature, it is meant to be set to be `-1`.
There might be other use cases to set it to be non `-1` when the current superset schema is not working properly in the future.